src/management/Akka.Management/AkkaManagementSetup.cs

```diff
@@ -0,0 +1,100 @@
+﻿// -----------------------------------------------------------------------
+//  <copyright file="AkkaManagementSetup.cs" company="Akka.NET Project">
+//      Copyright (C) 2009-2022 Lightbend Inc. <http://www.lightbend.com>
+//      Copyright (C) 2013-2022 .NET Foundation <https://github.com/akkadotnet/akka.net>
+//  </copyright>
+// -----------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using Akka.Actor.Setup;
+
+namespace Akka.Management
+{
+    public sealed class AkkaManagementSetup: Setup
+    {
+        public HttpSetup Http { get; set; }
+
+        internal AkkaManagementSettings Apply(AkkaManagementSettings settings)
+        {
+            return Http is null ? settings : new AkkaManagementSettings(Http.Apply(settings.Http));
+        }
+    }
+
+    public sealed class HttpSetup: Setup
+    {
+        /// <summary>
+        /// The hostname where the HTTP Server for Http Cluster Management will be started.
+        /// This defines the interface to use.
+        /// akka.remote.dot-netty.tcp.public-hostname is used if not overriden or empty.
+        /// if akka.remote.dot-netty.tcp.public-hostname is empty, <see cref="Dns.GetHostName"/> is used.
+        /// </summary>
+        public string Hostname { get; set; }
+        
+        /// <summary>
+        /// The port where the HTTP Server for Http Cluster Management will be bound.
+        /// The value will need to be from 0 to 65535.
+        /// </summary>
+        public int? Port { get; set; }
+        
+        /// <summary>
+        /// Use this setting to bind a network interface to a different hostname or ip
+        /// than the HTTP Server for Http Cluster Management.
+        /// Use "0.0.0.0" to bind to all interfaces.
+        /// </summary>
+        public string EffectiveBindHostname { get; set; }
```

Comment:
Want to stick with "public port" and "public hostname" conventions that we have established in Akka.Remote?